quantum/process_keycode/process_sequencer.c

```diff
@@ -0,0 +1,76 @@
+#include "process_sequencer.h"
+
+#if defined(SEQUENCER_ENABLE)
+
+bool process_sequencer(uint16_t keycode, keyrecord_t *record) {
```

Comment:
Lets make the spacing after each case statement to be consistent -- either remove the newlines after `return false;`, or add them in.